application-services/custom/camera-management/appcamera/routes.go

```diff
@@ -296,3 +305,26 @@ func (app *CameraManagementApp) ptzRoute(w http.ResponseWriter, req *http.Reques
 		app.lc.Error(err.Error())
 	}
 }
+
+func (app *CameraManagementApp) getPanTiltRange(deviceName string) (PanTiltRange, error) {
+	app.panTiltMutex.Lock()
+	defer app.panTiltMutex.Unlock()
+	val, exists := app.panTiltMap[deviceName]
+	if !exists {
+		fmt.Println("Device does not exist")
```

Comment:
we can probably remove this print

Reply:
Had just added for some testing :-) will remove